edx_exams/apps/lti/views.py

```diff
@@ -4,26 +4,39 @@
 
 from urllib.parse import urljoin
 
+from django.contrib.auth import login
 from django.http import JsonResponse
 from django.shortcuts import redirect
 from django.urls import reverse
 from django.views.decorators.http import require_http_methods
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
 from lti_consumer.api import get_end_assessment_return, get_lti_1p3_launch_start_url
 from lti_consumer.data import Lti1p3LaunchData, Lti1p3ProctoringLaunchData
 from lti_consumer.models import LtiConfiguration
+from rest_framework.decorators import authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
 
 from edx_exams.apps.core.models import ExamAttempt
 from edx_exams.apps.lti.utils import get_lti_root
 
+EDX_OAUTH_BACKEND = 'auth_backends.backends.EdXOAuth2'
+
 
 @require_http_methods(['GET'])
+@authentication_classes((JwtAuthentication,))
+@permission_classes((IsAuthenticated,))
 def start_proctoring(request, attempt_id):
     """
     LTI Start Proctoring
     """
     # TODO: Here we'd do all the start of proctoring things.
 
-    exam_attempt = ExamAttempt.objects.get(pk=attempt_id)
+    # user is authenticated via JWT so use that to create a
+    # session with this service's authentication backend
+    request.user.backend = EDX_OAUTH_BACKEND
+    login(request, request.user)
+
+    exam_attempt = ExamAttempt.objects.get(pk=attempt_id, user=request.user)
```

Comment:
added this very basic check here to prevent accidentally messing with another user's test attempt.